Cargo.toml

```diff
@@ -12,10 +12,12 @@ members = [
   "drivers",
   "drivers/test-fw",
   "fmc",
+  "fmc/test-rom",
   "hw-latest/verilated",
   "hw-model",
   "registers",
   "registers/bin/generator",
+  "rt",
```

Comment:
Is the idea that this would be shared between fmc and runtime firmware? Might be good to call this something else since there will also be a separate crate which implements runtime firmware.

Reply:
You are correct. What should we call it? I am very bad with names.

Reply:
we also briefly discussed earlier in the ROM/FMC standup call that maybe it makes more sense to rename @jhand2's module to something else like "App FW" instead of "Runtime FW" since the "rt" nomenclature is already used in rust developmnent. But if not, I suggest calling this one "Common" or something along those lines. I am fine with either direction, what do you think @jhand2?

Reply:
@jhand2
Jordan , I renamed it firmware crate.

Reply:
Either way is good with me (either renaming Runtime FW -> Application FW, or renaming this crate to just be named "firmware")

Reply:
I extracted the hand_off into a common crate and can rename the riscv-rt infrastructure support stuff to cpu , since the riscv stuff is not going to be used by rom. 

@jhand2  , would you like me to create an rt crate shell  as well ?

Reply:
> would you like me to create an rt crate shell as well ?

Ya, if you don't mind that would be great. Thanks!

Reply:
I have added the runtime crate.